common.blocks/progressbar/progressbar.bh.js

```diff
@@ -1,10 +1,18 @@
 module.exports = function(bh) {
 
     bh.match('progressbar', function(ctx, json) {
-        var val = json.val;
+        var val = json.val || 0;
 
         ctx
-            .js({ val : val })
+            .js({ val : json.val })
```

Comment:
:+1: for @sipayRT 

It's strange to have default 0 value in templates, while allow JS to fail.